apps/website/src/tailwind.css

```
@@ -1,66 +1,146 @@
 @import "tailwindcss";
 @plugin "daisyui" {
-	themes: night --default;
+  themes:
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)